Automattic/woocommerce-payments
# Revert "Update Connect page (#1600)"

Fixes (partially) #1612

This reverts commit fac29a8a1f4c61db44a787845085d725d055d644. This is to re-establish the original `< ConnectAccountPage  />` component for the A/B test in https://github.com/Automattic/woocommerce-payments/issues/1612. The previous changes in PR #1600 will be added to the A/B test as a new component.

#### Testing instructions

1. Navigate to Payments
2. See that that the original `<ConnectAccountPage />` component renders.

![image](https://user-images.githubusercontent.com/9312929/116870129-80960100-ac44-11eb-9c3d-5394c9a6bd22.png)
